neo_force_scheme/pca.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+import time
+import numpy as np
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from sklearn.decomposition import PCA
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+import seaborn as sns
```

Comment:
its better to remove seaborn from here because we don't include seaborn in our dependency (requirements.txt)